src/hooks/useAdminApi.ts

```diff
@@ -30,12 +34,21 @@ export const useAdminApi = () => {
   const updateEpochsMap = useSetRecoilState(rEpochsRaw);
   const updateUsersMap = useSetRecoilState(rUsersMapRaw);
 
-  // A fake circleId will just return nothing
-  const selectedCircleId = useRecoilValue(rSelectedCircleId) ?? -1;
-  const myAddress = useRecoilValue(rMyAddress);
+  const getState = useRecoilCallback(
```

Comment:
interesting. so `useAdminApi()` for some reason isn't refreshing these two values when you change your address or the circle id? that seems weird... i wonder if there's a fix that gets closer to the root issue than this?

Reply:
not a blocker, just thinking out loud

Reply:
The problem was actually that it was being set on the column's useMemo and those didn't update. The underlying dependency isn't necessarily available either. I think it is best for these functions be be stateless, but perhaps what I am doing here is still odd...